web/src/test/manual/embed/android-harness/build.sh

```diff
@@ -21,6 +21,10 @@ cp "$KEYMAN_ROOT/web/build/app/embed/debug/osk/kmwosk.css" "$THIS_DIR/host/kmwos
 cp "$KEYMAN_ROOT/web/build/app/embed/debug/osk/globe-hint.css" "$THIS_DIR/host/globe-hint.css"
 cp "$KEYMAN_ROOT/web/build/app/embed/debug/osk/keymanweb-osk.ttf" "$THIS_DIR/host/keymanweb-osk.ttf"
 
+# We shouldn't rely on a prior Android build for these files!  We can retrieve them separately.
+cp "$KEYMAN_ROOT/common/web/sentry-manager/build/index.js" "$THIS_DIR/host/keyman-sentry.js"
+cp "$KEYMAN_ROOT/node_modules/es6-shim/es6-shim.min.js" "$THIS_DIR/host/es6-shim.min.js"
```

Comment:
While these two files aren't particularly necessary for direct functionality for the Android-harness test page, the errors that occur when trying to load and use them - `keyman-sentry.js`, in particular - seem to cascade into other errors that bring the whole house of cards down.

Worth noting:  up until this PR chain, we never actually 'published' this testing page on the testing index.  Looking up an old PR's artifacts, this didn't work before, either.  That said... I think a little "elbow grease" can overcome this and get this page operational for future user tests.